prow/scripts/cluster-integration/kyma-gke-upgrade.sh

```diff
@@ -68,463 +79,399 @@ export EXTERNAL_SOLUTION_TEST_RELEASE_NAME="${EXTERNAL_SOLUTION_TEST_NAMESPACE}"
+    curl -L --silent --fail --show-error "https://github.com/kyma-project/kyma/releases/download/${LAST_RELEASE_VERSION}/kyma-installer-cr-cluster.yaml" --output /tmp/kyma-gke-upgradeability/kyma-installer-cr-cluster.yaml
 
-    shout "Installation triggered with timeout ${KYMA_INSTALL_TIMEOUT}"
-    date
-    "${KYMA_SCRIPTS_DIR}"/is-installed.sh --timeout ${KYMA_INSTALL_TIMEOUT}
-}
+    kubectl apply -f /tmp/kyma-gke-upgradeability/kyma-installer.yaml
+    kubectl apply -f /tmp/kyma-gke-upgradeability/kyma-installer-cr-cluster.yaml
+  fi
 
-function checkTestPodTerminated() {
-    local namespace=$1
-    local retry=0
-    local runningPods=0
-    local succeededPods=0
-    local failedPods=0
-
-    while [[ "${retry}" -lt "${TEST_TIMEOUT_SEC}" ]]; do
-        # check status phase for each testing pods
-        for podName in $(kubectl get pods -n "${namespace}" -o json | jq -sr '.[]|.items[].metadata.name')
-        do
-            runningPods=$((runningPods + 1))
-            phase=$(kubectl get pod "${podName}" -n "${namespace}" -o json | jq '.status.phase')
-            echo "Test pod '${podName}' has phase: ${phase}"
-
-            if [[ "${phase}" == *"Succeeded"* ]]
-            then
-                succeededPods=$((succeededPods + 1))
-            fi
-
-            if [[ "${phase}" == *"Failed"* ]]; then
-                failedPods=$((failedPods + 1))
-            fi
-        done
-
-        # exit permanently if one of cluster has failed status
-        if [[ "${failedPods}" -gt 0 ]]
-        then
-            echo "${failedPods} pod(s) has failed status"
-            return 1
-        fi
-
-        # exit from function if each pod has succeeded status
-        if [[ "${runningPods}" == "${succeededPods}" ]]
-        then
-            echo "All pods in ${namespace} namespace have succeeded phase"
-            return 0
-        fi
-
-        # reset all counters and rerun checking
-        delta=$((runningPods-succeededPods))
-        echo "${delta} pod(s) in ${namespace} namespace have not terminated phase. Retry checking."
-        runningPods=0
-        succeededPods=0
-        retry=$((retry + 1))
-        sleep 5
-    done
-
-    echo "The maximum number of attempts: ${retry} has been reached"
-    return 1
+  log::banner "Installation triggered with timeout ${KYMA_INSTALL_TIMEOUT}"
+  "${KYMA_SCRIPTS_DIR}"/is-installed.sh --timeout ${KYMA_INSTALL_TIMEOUT}
 }
 
 function installTestChartOrFail() {
   local path=$1
   local name=$2
   local namespace=$3
 
-  shout "Create ${name} resources"
-  date
+  log::info "Create ${name} resources"
 
   helm install "${name}" \
-      --namespace "${namespace}" \
-      --create-namespace \
-      "${path}" \
-      --timeout "${HELM_TIMEOUT_SEC}" \
-      --set domain="${DOMAIN}" \
-      --wait
+    --namespace "${namespace}" \
```

Comment:
Why you deleted 2 spaces?

Reply:
the reason is that shfmt reformatted the file and this is what it came up with. Settings are similar to what google suggests: https://google.github.io/styleguide/shellguide.html